test/unit/lifecycle/properties.js

```diff
@@ -219,35 +219,98 @@ describe('lifecycle/properties', function () {
     expect(el.getAttribute('prop1')).to.equal('');
   });
 
-  describe('270', function () {
-    function setup (initialValue) {
-      var el = elem.create();
-      el.textContent = 'existing content';
+  it('should not be triggered on initialisation', function () {
+    let triggered;
+    let newValue;
+    let oldValue;
+
+    skate(elem.safe, {
+      properties: {
+        prop: {
+          update: function (nv, ov) {
+            triggered = true;
+            newValue = nv;
+            oldValue = ov;
+          }
+        }
+      }
+    });
+
+    let el = elem.create();
+    expect(triggered).to.equal(undefined);
+    expect(newValue).to.equal(undefined);
+    expect(oldValue).to.equal(undefined);
+
+    el.prop = 'one';
+    expect(triggered).to.equal(true);
+    expect(newValue).to.equal('one');
+    expect(oldValue).to.equal(undefined);
+
+    el.prop = 'two';
+    expect(triggered).to.equal(true);
+    expect(newValue).to.equal('two');
+    expect(oldValue).to.equal('one');
+  });
+
+  it('should override existing properties', function () {
+    skate(elem.safe, {
+      properties: {
+        textContent: {}
+      }
+    });
+
+    let el = skate.create(`<${elem.safe}>initial content</${elem.safe}>`);
+    expect(el.textContent).to.equal('initial content');
+    el.textContent = 'updated content';
+    expect(el.textContent).to.equal('updated content');
+    expect(el.innerHTML).to.equal('initial content');
+  });
+
+  describe('templating integration', function () {
```

Comment:
@nicolasv, @cyberdash these are the common scenarios we discussed. We didn't discuss what happens if you override `innerHTML`, though. The skinny of it is that if you override `innerHTML` then you can't use it for templating in scenario 2, however, it will still work in scenario 1. In scenario 2, though, we'd probably use a dom differ of some sort anyways which wouldn't be using innerHTML.